content/2021/2021-01-21_updates-from-the-past-year/index.md

```diff
@@ -0,0 +1,49 @@
++++
+
+
+The year 2020 and the beginning of 2021 was a very busy period for Codewars maintainers. We worked hard on improving the platform, and on fixing existing bugs that were getting in a way of further improvements. We were looking for ways and ideas on how to get closer to Codewars users. At the same time, we managed to introduce a few new languages to train on, upgraded some language versions, and provided some means to support users and newcomers.
+
+
+## New blog
+
+We migrated the Codewars blog to a new platform, and plan to use it for announcements for the Codewars community. We will post here information and updates about new features, improvements, and we will call for your help when we need one. Blog posts will show up on the dashboard of the main site, which will hopefully help to bring the Codewars closer to its users.
+
+
+## New languages
+
+2020 brought a large set of new languages which users can train on, create new kata, and translate existing kata to! Most of them are still in the beta stage, and contributions in form of new kata or translations will be very helpful. Many languages were also updated to new versions. Some of the added languages are Lean, COBOL, Perl, Raku, and Pascal, but these are not the only ones! More details will be provided in a dedicated post.
+
+Many thanks to all users who provided their support on setting up the code runner and created new content for introduced languages!
+
+
+## Refactoring and improvements
+
+Refactoring was probably the biggest topic of all activities related to Codewars conducted in recent time. Thanks to code cleanup, reducing technical debt, and even total rewrite of some elements, it was possible to improve performance in the backend and improve user experience and navigation on the frontend. Getting rid of some messy code also enabled maintainers to provide further improvements in the future, which, until now, were very difficult to add.
+
+
+## Documentation
+
+Addressing the needs of new users and questions of many experienced ones, a new [documentation site](https://docs.codewars.com/) has been launched. Documentation is meant to be created by the community, so everyone can write articles, review drafts, suggest improvements, or fix any issues using [codewars/docs](https://github.com/codewars/docs) GitHub repository. The documentation is still a work in progress, but when it gets into an acceptably good shape, it will replace the old Codewars wiki.
+
+More information on documentation will be provided in a dedicated announcement. Every contribution is welcome!
+
+
+## Community moderation team
+
+To take some maintenance burden off of the administrators' back, the [Codewars Community Moderation Team](https://docs.codewars.com/community/moderation/#members) has been formed and provided with a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```

Comment:
```suggestion
The Codewars community needed moderation for a long time, but it was difficult because we only have one developer/admin. We wanted to delegate various tasks to a team of trusted users, but all the technical issues got in the way...

We're excited to introduce [the community moderation team](https://docs.codewars.com/community/moderation/#members)! They have access to a variety of tools to manage content, handle problematic users, support users, and perform other moderation tasks. 
```